ministryofjustice/opg-refunds
# Lpa 2684 pr test (#2)

* LPA-2684: Formatting and Jenkinsfile update

- Changing agent target as advised by ops
- Small formatting change to test triggers
- Tidy up Jenkins file to trigger Jenkins
- Remove functions we no longer use.
- Remove makefile and make target we don't use.
- Make j2_cleanup_pipeline_run isn't in the makefile and we don't what it
should do above anyway
